src/elements/overlay/overlay.visual.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { html, type TemplateResult } from 'lit';
+
+import { describeViewports, visualDiffDefault } from '../core/testing/private.js';
+
+import './overlay.js';
+import type { SbbOverlayElement } from './overlay.js';
+
+describe(`sbb-overlay`, () => {
+  const defaultArgs = {
+    expanded: false,
+    backButton: false,
+    negative: false,
+  };
+
+  const template = ({ expanded, backButton, negative }: typeof defaultArgs): TemplateResult => html`
+    <sbb-overlay ?negative=${negative} ?expanded=${expanded} ?back-button=${backButton}>
+      <div class="overlay-content">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+        labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+        laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+        voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+        cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+      </div>
+    </sbb-overlay>
+  `;
+
+  const openOverlay = (setup: any): void => {
+    const overlay: SbbOverlayElement = setup.snapshotElement.querySelector('sbb-overlay')!;
+    setup.withSnapshotElement(overlay);
+    setup.withPostSetupAction(() => overlay.open());
+  };
+
+  describeViewports({ viewportHeight: 600 }, () => {
```

Comment:
does it make sense to test every viewport, or would a subset be fine?

Reply:
On Figma i see every viewport, but if required we can reduce it (eg. zero/small/large/ultra ?)

Reply:
I found styles for zero, small, medium, large and ultra. so maybe keeping all is ok ;-)